### ISSUE TITLE
Provide option to update a single contribution in CM

### ISSUE THREAD

**Issue by Akarshit** [processing/processing#4242]
Right now the Updates tab in the contribution manager shows all the contribution for which the updates are available but provides only the option to update all contributions together through a Update All button.
If someone has to update only a single contribution he would have to go to the respective tab and then search for the contribution and then update it. This is a rather long procedure and something similar should be provided.
I suggest providing a update button in a separate column with every contribution in the Updates tab so that even if the user wants to update a single contribution it can be done easily.

@alignedleft What is your input on this ?

**Comment by iamutkarshtiwari** [comment]
@Akarshit Would you like me to work on this?

**Comment by Akarshit** [comment]
@iamutkarshtiwari It would be nice to first get this approved by @alignedleft as this is just a idea that popped into my mind. Also if this gets approved, the UI need to be designed and i think that area is the expertise of @jamesjgrady

**Comment by iamutkarshtiwari** [comment]
@Akarshit I don't think that we need to do some major changes to the UI. All we can do is just make the 'exclamatory(update available)' icons in front of each contribution clickable so that each contribution can be updated indivisually depending upon user's choice. @benfry What would you suggest?
